themes/default/content/blog/pulumiup-google-native-provider/index.md

```diff
@@ -107,21 +119,33 @@ func main() {
 ```csharp
 using System.Threading.Tasks;
 using Pulumi;
-using Pulumi.GoogleNative;
+using Pulumi.GoogleNative.Container.V1;
+
 class Program
 {
-    static Task Main() =>
-        Deployment.Run(() => {
-            var bucket = new GoogleNative.Storage.v1.Bucket("my-bucket");
+    static Task<int> Main() => Deployment.RunAsync(() =>
+    {
+        var project = "mythical-ember-311422";
+        var region = "us-central1";
+        var clusterName = "gke-native";
+
+        var cluster = new Cluster("cluster", new ClusterArgs
+        {
+            ProjectsId = project,
+            LocationsId = region,
+            ClustersId = clusterName,
+            Name = clusterName,
+            Autopilot = new Pulumi.GoogleNative.Container.V1.Inputs.AutopilotArgs { Enabled = true }
```

Comment:
Can this be shortened given the using above?

Reply:
Add another `using Pulumi.GoogleNative.Container.V1.Inputs;` and then just use `AutopilotArgs` here